### PR TITLE
Update gaussian_naive_bayes.py

### PR DESCRIPTION
**Pull request by mvsg2**
Using the seaborn.heatmap library to plot the confusion matrix instead of the plot_confusion_matrix method from the sklearn.metrics module.

### Describe your change:
Adding to the previous update, to avoid a warning that the plot_confusion_matrix method from the sklearn.metrics throws, I have used the heatmap module of seaborn which takes an argument - the confusion matrix obtained from the confusion_matrix method in the sklearn.metrics library.

* [ ] Add an algorithm?
* [x] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### Checklist:
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: machine_learning/gaussian_naive_bayes.py [hunk 1]
```diff
@@ -1,11 +1,15 @@
...
 from sklearn.datasets import load_iris
-from sklearn.metrics import accuracy_score, confusion_matrix # The plot_confusion_matrix method will be deprecated in the developed versions of Python 3.10.x according to the warning that it throws when this code is run.
+from sklearn.metrics import (  # The plot_confusion_matrix method will be deprecated in the developed versions of Python 3.10.x according to the warning that it throws when this code is run.
+    accuracy_score,
+    confusion_matrix,
+)
 from sklearn.model_selection import train_test_split
 from sklearn.naive_bayes import GaussianNB
-import time
-import seaborn as sns # For plotting a heatmap of the confusion matrix. This way will not throw any warning.
```
Comment (mvsg2) [inline machine_learning/gaussian_naive_bayes.py]: Sorry, didn't mean to import them again...

**Comment by cclauss** [comment]
Please read CONTRIBUTING.md.  Algorithmic functions should return values and they should not print or plot.

**Comment by ZeroDayOwl** [comment]
@mvsg2 you have duplicate PR #7406. Close one.

File: machine_learning/gaussian_naive_bayes.py [hunk 2]
```diff
@@ -25,20 +31,22 @@ def main():
...
+    print("The confusion matrix is:\n", Conf_Matrix)
+
+    time.sleep(1.8)
+    # Declaring the overall accuracy of the model
+    final_accuracy = 100 * accuracy_score(y_true=y_test, y_pred=y_pred)
+    print(f"The final accuracy of the model is: {round(final_accuracy, 2)}%")
+
 
 if __name__ == "__main__":
     main()
```
Comment (ZeroDayOwl) [inline machine_learning/gaussian_naive_bayes.py:52]: Perhaps add doctest cases.

**Review by ZeroDayOwl — CHANGES_REQUESTED** [review]

**Pull request closed without merging**